miner/miner_test.go

```diff
@@ -152,7 +152,6 @@ func createMiner(t *testing.T) *Miner {
 	if err != nil {
 		t.Fatalf("can't create new chain config: %v", err)
 	}
-
```

Comment:
unnecessary diff?

Reply:
Matches the upstream code I believe. So it reduces the diff in the op-geth repo.